domains/_vercel.kushdhingra.json

```diff
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "username": "kushdhingra"
```

Comment:
Username is different from the main file. You might want to consider changing the main files to the same username so that checks pass.